api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -318,10 +318,7 @@ def test_not_published(self):
         res = self.app.get(self.published_preprint_url)
         assert res.status_code == 200
         citation = res.json['data']['attributes']['citation']
-        assert citation == 'McGee, Grapes C. B. “{}.” {}, {} Web.'.format(
-            self.published_preprint.title,
-            self.published_preprint.provider.name,
-            'n.d.')
+        assert citation == f'McGee, Grapes C. B. “{self.published_preprint.title}.” {self.published_preprint.provider.name}, n.d. Web.'
```

Comment:
This citations used the article_doi, which was incorrect and just produced an empty string. That was still allowing the tests to pass. Now that I fixed article_doi, so it returns a valid DOI or `None` that cause these strings to break (cannot concatinate None with sting), so updating these tests and using F-strings